task-definition.plugins.json

```diff
@@ -164,8 +164,8 @@
             },
             "healthCheck": {
                 "retries": 10,
-                "command": ["CMD-SHELL", "curl -f http://localhost:6738/_ready || exit 1"],
-                "timeout": 10,
+                "command": ["CMD-SHELL", "curl -f http://localhost:6738/_health || exit 1"],
```

Comment:
wasn't there a reason we couldn't use `_health` and needed to use `_ready`?

Reply:
Hmm we were using this as a readiness check but now I've extended it to be more of a healthcheck. The previous healthcheck was also terrible so I just got rid of it